lib/units/ios-device/plugins/info/index.js

```diff
@@ -16,9 +16,10 @@ module.exports = syrup.serial()
         let solo = wireutil.makePrivateChannel()
 
         let osName = "iOS"
-        if (options.deviceType === "tvOS") {
+        if (options.deviceName.includes('TV') || options.deviceName.includes('tv')) {
```

Comment:
how bout converting to lowercase and compare with `tv` only?

Reply:
Great observation, will submit another change for that.